src/GrpcProtos/capture.proto

```diff
@@ -120,15 +120,20 @@ message InternedTracepointInfo {
   TracepointInfo intern = 2;
 }
 
-message TracepointEvent {
+message InternedTracepointEvent {
```

Comment:
Same problem with naming here as with `InternedCallstackSample`. "Interned" here means something different than "Interned" in `InternedTracepointInfo` or `InternedTracepointInfo`. Either drop "Interned" altogether or change prefix.